tools/harness-thci/OpenThread.py

```diff
@@ -1204,6 +1204,10 @@ def joinNetwork(self, eRoleId):
                 print('join as sleepy end device')
                 mode = '-'
                 self.__setPollPeriod(self.__sedPollPeriod)
+            elif eRoleId == Thread_Device_Role.SSED:
+                print('join as SSED')
+                mode = '-'
```

Comment:
Question: Why didn't we use `n` mode for SED/SSED?
@librasungirl

Reply:
I guess requiring stable network data only is a requirement for reference devices.